public/app/features/plugins/components/AppRootPage.test.tsx

```diff
@@ -72,13 +73,13 @@ function renderUnderRouter() {
   const route = {
     component: () => <AppRootPage pluginId="my-awesome-plugin" pluginNavSection={appsSection} />,
   } as unknown as RouteDescriptor;
-  locationService.push('/a/my-awesome-plugin');
+  locationService.push(`/a/my-awesome-plugin/${page}`);
```

Comment:
if `$page` is undefined this will print `/a/my-awesome-plugin/undefined`

Reply:
This is a good point, we probably should fall back to the default plugin page routes in case page is not defined (and in that case the existing tests would keep testing the entry route without any page).

E.g.:
```typescript
const baseUrl = '/a/my-awesome-plugin';
const url = page ? `${baseUrl}/${page}` : baseUrl;

const baseRoute = '/a/:pluginId';
const route = page ? `${baseRoute}/${page}` : baseRoute;
```